Classes/MOC/Varnish/Service/VarnishBanService.php

```diff
@@ -93,6 +93,9 @@ public function banByTags(array $tags, $domain = NULL) {
 		if (count($this->settings['ignoredCacheTags']) > 0) {
 			$tags = array_diff($tags, $this->settings['ignoredCacheTags']);
 		}
+		$tags = str_replace(':', '-', $tags);
+		$tags = str_replace('.', '_', $tags);
```

Comment:
Hey Christoph thanks for fixing this one. However I'd prefer if it was fixed the same way it's done in the core https://github.com/neos/typoscript/blob/master/Classes/TYPO3/TypoScript/Core/Cache/ContentCache.php#L356

we also need to fix it in the 1.0 branch